channels/chan_usrp.c

```diff
@@ -662,6 +666,7 @@ static int usrp_xwrite(struct ast_channel *ast, struct ast_frame *frame)
 		fr.delivery.tv_sec = 0;
 		fr.delivery.tv_usec = 0;
 		ast_queue_frame(ast, &fr);
+		ast_debug(1, "Channel %s: RX OFF\n", ast_channel_name(ast));
```

Comment:
Won't this debug message get triggered for every single frame processed?

Reply:
This only occurs when the rxkey variable counts down to 1.  The code that follows will decrement it to zero.  Previous code sets it to a number above 1 so that this action only occurs one time.